### PR TITLE
Deprecate chunks

### LINKED ISSUE #72: Deprecate chunks in teal.code

### PR DESCRIPTION
**Pull request by nikolas-burkoff**
Closes #72

### CODE REVIEW

File: vignettes/advanced_chunks.Rmd [hunk 1]
```diff
@@ -9,6 +9,8 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+NOTE: Chunks are being deprecated `qenv` objects should be used instead - new vignettes are on their way.
```
Comment (asbates) [inline vignettes/advanced_chunks.Rmd:12]: I would think about making these notes stand out a bit more. Maybe make 'NOTE' bold.

Comment (nikolas-burkoff) [reply vignettes/advanced_chunks.Rmd:12]: True - although before we merge into main the plan is to remove these vignettes completely and write new ones for qenv -> see [here](https://github.com/insightsengineering/teal/issues/731) 

image

So this was a temporary fix to make the teal_refactor branch sort of correct

File: _pkgdown.yml [hunk 2]
```diff
@@ -28,3 +18,14 @@ reference:
...
+    desc: Functions to create a code chunk stack
+    contents:
+      - contains("chunk")
+  - title: "Debugging chunks evaluation (deprecated)"
+    desc: "shiny modules to examine chunks evaluation results"
+    contents:
+      - get_eval_details_srv
+      - get_eval_details_ui
+      - show_eval_details_modal
+
```
Comment (nikolas-burkoff) [inline _pkgdown.yml:31]:
```suggestion
      
```

Comment (nikolas-burkoff) [inline _pkgdown.yml:31]:
```suggestion
```

**Review by mhallal1 — APPROVED** [review]
Screenshot from 2022-09-29 15-19-43

**Pull request merged**